app/src/main/java/com/github/libretube/util/TextUtils.kt

```diff
@@ -111,30 +110,21 @@ object TextUtils {
         else -> null
     }
 
-    fun formatRelativeDate(context: Context, unixTime: Long): CharSequence {
+    fun formatRelativeDate(unixTime: Long): CharSequence {
         val date = LocalDateTime.ofInstant(Instant.ofEpochMilli(unixTime), ZoneId.systemDefault())
         val now = LocalDateTime.now()
         val months = date.until(now, ChronoUnit.MONTHS)
 
         return if (months > 0) {
             val years = months / 12
 
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-                val (timeFormat, time) = if (years > 0) {
-                    RelativeDateTimeFormatter.RelativeUnit.YEARS to years
-                } else {
-                    RelativeDateTimeFormatter.RelativeUnit.MONTHS to months
-                }
-                RelativeDateTimeFormatter.getInstance()
-                    .format(time.toDouble(), RelativeDateTimeFormatter.Direction.LAST, timeFormat)
+            val (timeFormat, time) = if (years > 0) {
+                RelativeDateTimeFormatter.RelativeUnit.YEARS to years
             } else {
-                val (timeAgoRes, time) = if (years > 0) {
-                    R.plurals.years_ago to years
-                } else {
-                    R.plurals.months_ago to months
```

Comment:
Same for this plural

Reply:
I agree, that's reasonable.

They're probably not yet translated in all languages, so it makes sense to prevent translators for spending time with translating them.